src/map/ai/controllers/mob_controller.cpp

```diff
@@ -1264,6 +1302,30 @@ bool CMobController::Cast(uint16 targid, SpellID spellid)
     return CController::Cast(targid, spellid);
 }
 
+void CMobController::SetFollowTarget(CBaseEntity* PTarget, FOLLOWTYPE followType)
```

Comment:
Would it make sense to default one or both of these to nullptr and None? Setting the target to nullptr AND the follow type to None in the same function seems a little smelly, any other ideas?